### DISCUSSION TITLE
yunohost LXC behind Proxy manager LXC

### DISCUSSION THREAD

**Discussion by protonaut** [tteck/Proxmox#3174] [Help Needed]
I just tried to install the yunohost LXC and was wondering how to configure it for access from outside, which means for me behind a nginxproxymanager. 

I'd used yunohost on baremetal for quite some time and this setting worked as expected. I remember from the installation instructions that it is strongly adviced to use yunohost without any reverse proxy.
So the first try to serve yunohost through the proxymanager just failed instantly.

So, any advice, hints or links how to configure?

**Comment by Mellowlynx** [comment]
How did you set it up within the proxy manager?
Sometimes setting up something behind a reverse proxy can be a pain indeed.

**Reply by protonaut** [reply]
I installed nginxproxymanager und yunohost separatly with the scripts  tteck provided. 
Within the proxymanager I wanted to create a host for a subdomain  to point directly to the host where yunohost is located.

When I created it with an SSL-certificate (letsencrypt) I got an internal error within the proxymanager. 
Without the SSL_certificate the browser struggled to connect.
